streamlineTools/adminCommerceContent.js

```
@@ -1,11 +1,27 @@
-let testCode = ''
+// let code = ''
+// let testCode = ''
 // var filename = filename || "";
-let filename
-let filenameAfter
-let filenameBefore
+// let filename
+// let filenameAfter
+// let filenameBefore
+
+if (typeof code === "undefined") {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)